README.md

```diff
@@ -267,17 +267,27 @@ For linking and running the game, the following libraries are also required:
    * or start it directly  
      `./game/ultrastardx`
 
-#### Compiling on OS X
+#### Compiling on MacOS (High Sierra)
 - USDX is built using _Homebrew_ and official _FreePascal build_ (using its compiler _FPC_)
 - You can install Homebrew from [brew.sh](http://brew.sh)
-- You can get the FPC build from [freepascal.org](http://www.freepascal.org/down/i386/macosx.var)
+- You can get the FPC build from [freepascal.org](http://www.freepascal.org/down/i386/macosx.var) or
+  * `brew install fpc`
 - Don't miss _XQuartz_ from [xquartz.org](http://www.xquartz.org)
-- Make sure the XCode command line tools are installed. `xcode-select --install`
+- Make sure the XCode command line tools are installed. 
+  * `xcode-select --install`
 - Needed brew libraries can be installed using:
-  * `brew install sdl2 sdl2_image ffmpeg libav portaudio binutils sqlite freetype lua libtiff`
-  * `brew switch ffmpeg 2.8.6`
-- Pass argument `--enable-osx-fink` or `--enable-osx-brew` (default) according to the packaging you are using
-- `./configure`
-- `make macosx-standalone-app`
+  * `brew install sdl2 sdl2_image ffmpeg@2.8 libav portaudio binutils sqlite freetype lua libtiff`
+  * `export PATH=/usr/local/opt/ffmpeg@2.8/bin:"$PATH"`
```

Comment:
I doubt this command is needed

Reply:
When you install ffmpeg@2.8 it gives you that instruction. It is not added to the path automatically since 3.4 is the supported version.

Reply:
But I'm pretty sure we don't need ffmpeg to be in the path.
We only need its .dylib and .pc files.